### DISCUSSION TITLE
Ayuda

### DISCUSSION THREAD

**Discussion by Francolin09** [Launch-X-Latam/MisionFrontEnd#565] [Team Posts]
Hola chicos, no encuentro el acceso a la misión bakcend por lo que solicito ayuda, además me encuentro en busqueda de grupo para la hackaton por si a alguien le falta algun miembro

**Comment by IsraelVera** [comment]
Agrégame si se puede yo no tengo equipo ***@***.***

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: Francolin09 ***@***.***>
Sent: Tuesday, November 15, 2022 10:38:19 AM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-backend] Ayuda (#102)


Hola chicos, no encuentro el acceso a la misión bakcend por lo que solicito ayuda, además me encuentro en busqueda de grupo para la hackaton por si a alguien le falta algun miembro

**Comment by xramons** [comment]
https://teams.microsoft.com/l/team/19%3a59Tb8L_0PlM5uxQtC8AjVfcF-wvYCKBxsS1JRWZLnyo1%40thread.tacv2/conversations?groupId=e6804a94-ce40-41ca-a5c2-b287fc50036b&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816
